src/mcl_3dl.cpp

```diff
@@ -109,42 +112,49 @@ class MCL3dlNode
   void cbMapcloud(const sensor_msgs::PointCloud2::ConstPtr& msg)
   {
     ROS_INFO("map received");
-    pcl::PointCloud<pcl::PointXYZI> pc_tmp;
-    pcl::fromROSMsg(*msg, pc_tmp);
-
-    if (pc_tmp.points.size() == 0)
+    pcl::PointCloud<pcl::PointXYZI>::Ptr pc_tmp_raw(new pcl::PointCloud<pcl::PointXYZI>);
+    pcl::fromROSMsg(*msg, *pc_tmp_raw);
```

Comment:
switch by input point type to remove warnings (`Failed to find match for field 'intensity'.`).